### PR TITLE
Ask VA 996 - Design Changes for Check the status of your question - "Check status of your question"

### PR DESCRIPTION
**Pull request by hemeshvpatel**
## Summary

- The Check status of your question UI is updated on the introduction page per [figma](https://www.figma.com/design/aQ6JsjD4pvMxSVPAZHllMX/AVA-Page-Library?node-id=1224-131180&t=L2IzQCKenMG1LGJL-0)
- Mock response is put in until endpoint is finished 
- New, In progress, Solved, and Reopened statuses are added
- The Ask VA team owns this form
- This is part of our POC

## Related issue(s)

- Link to ticket [here](https://app.zenhub.com/workspaces/ask-va-647a476551689d06655cc815/issues/gh/department-of-veterans-affairs/ask-va/996)

## Testing done

- Existing tests

## Screenshots

Not found:

image

Question was submitted as an Authenticated user:

image

image

image

image

Question was submitted as an UnAuthenticated user example:

image

## What areas of the site does it impact?

- This only impacts the Ask VA form

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

File: src/applications/ask-va/containers/IntroductionPage.jsx [hunk 1]
```diff
@@ -33,13 +39,127 @@ const IntroductionPage = props => {
...
     [props],
   );
 
+  // const getApiData = url => {
+  //   return apiRequest(url)
+  //     .then(() => {
+  //       setInquiryData(res);
+  //     })
+  //     .catch(() => setHasError(true));
+  // };
```
Comment (hemeshvpatel) [inline src/applications/ask-va/containers/IntroductionPage.jsx:59]: Temporarily commented out until we have the endpoint finalized

File: src/applications/ask-va/containers/IntroductionPage.jsx [hunk 2]
```diff
@@ -33,13 +39,127 @@ const IntroductionPage = props => {
...
+
+  const handleSearchByReferenceNumber = () => {
+    // const url = `${
+    //   environment.API_URL
+    // }/inquiries/${searchReferenceNumber}/status`;
+    // getApiData(url);
```
Comment (hemeshvpatel) [inline src/applications/ask-va/containers/IntroductionPage.jsx:65]: Temporarily commented out until we have the endpoint finalized

**Review by Khoa-V-Nguyen — DISMISSED** [review]
lgtm

**Review by Khoa-V-Nguyen — APPROVED** [review]
lgtm

**Review by ediiotero — APPROVED** [review]
looks good. At some point we will need to change these platform paths to @department-fo-veterans-affairs

**Review by ataker — APPROVED** [review]

**Pull request merged**